.appveyor.yml

```
@@ -55,6 +55,7 @@ install:
   - conda config --set always_yes true
   - conda config --set show_channel_urls yes
   - conda config --prepend channels conda-forge
+  - conda update -n base -q conda
```

```suggestion
  - conda install python=3.11
  - conda update -n base -q conda
```
Perhaps this would work to get a newer version of Python?